### PR TITLE
Winter 2023 Main page updates

### PR DESCRIPTION
**Pull request by avahoffman**

### CODE REVIEW

**Comment by avahoffman** [comment]
OOf actions is hanging on the pdf rendering. Will investigate 😬

**Review by carriewright11 — APPROVED** [review]
Looks good! Lol you beat me to it! I sent an email asking about when grades will be due, since there was one time that it was earlier than expected, just to make sure the due date will be OK.

File: materials_schedule.Rmd [hunk 1]
```diff
@@ -4,7 +4,7 @@ output:
   html_document
 ---
 
-In recognition of [Juneteenth](https://en.wikipedia.org/wiki/Juneteenth), there will be no class on Monday June 20th 2022. There is an assignment due before class to [install software](../../modules/Setup/R_and_RStudio_installation.html) and we recommend completing HW 1 (uploading a screenshot showing that you finished the Dataquest module indicated below) before class, but all other assignments will be due July 1st. You are welcome to turn assignments in earlier if you wish.
+In recognition of [Martin Luther King Jr. Day](https://en.wikipedia.org/wiki/Martin_Luther_King_Jr._Day), there will be no class on Monday January 16th, 2023. There is an assignment due before class to [install software](../../modules/Setup/R_and_RStudio_installation.html) and we recommend completing HW 1 (uploading a screenshot showing that you finished the Dataquest module indicated below) before class, but all other assignments will be due January 30th. You are welcome to turn assignments in earlier if you wish.
```
Comment (carriewright11) [inline materials_schedule.Rmd:7]: could add the time for the due date?

**Comment by carriewright11** [comment]
oh we also need to update homework 0 resources - I can add that here

**Comment by carriewright11** [comment]
Looks like the git action isn't working in general - not just for us: https://github.com/fifsky/html-to-pdf-action/actions/runs/3579748210/jobs/6021240703 - might be because some aspects are out-of-date- https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md

maybe we can use another option

**Comment by carriewright11** [comment]
this isn't a one to one type of action - I see others but they don't have as many stars - but this one has many stars and could do more of the rendering: https://github.com/marketplace/actions/markdown-to-pdf-and-html

**Comment by carriewright11** [comment]
otherwise could try this: https://github.com/marketplace/actions/convert-html-to-pdf

**Comment by carriewright11** [comment]
hmmm the rendering is still using the old action:
Screen Shot 2022-12-24 at 11 59 03 AM

I am going to merge this to see what happens

**Pull request merged**